src/notation/internal/notationconfiguration.cpp

```diff
@@ -266,7 +266,7 @@ const QPixmap& NotationConfiguration::backgroundWallpaper() const
 
     if (path.empty()) {
         wallpaper = QPixmap();
-    } else if (path != lastPath) {
+    } else if (path != lastPath || wallpaper.isNull()) {
```

Comment:
An alternative solution might be to initialise `lastPath` with an empty path instead of `path`, so that `path != lastPath` is true.
It seems to me that the disadvantage of checking `wallpaper.isNull()` is that if a wallpaper fails to load, we'll retry loading it every time this method is called, which is wasteful.

Reply:
That makes sense. Failing to load hopefully is a rare case, but I'll do a force push with a static default initialization of lastPath - the code already updates last path afterwards. 

P.S. Speaking of wasteful, it would be cool if you could give an examination of the problem over at issue #12753 (few second delay for search activation when not currently searching). Off topic here, but thought I'd mention it in passing. The "front-end" QML seems okay to me -  but something screwy going on with the architecture behind it or something.